src/components/PushNotifications/index.js

```diff
@@ -54,15 +55,14 @@ const createPushNotifications = ({
       sendPushSubscription: {
         run: async () => {
           if (!isComponentConfigured(config)) {
-            throw new Error(
-              "Push notifications module is not configured. No VAPID public key was provided.",
-            );
+            throw new Error("Push notifications module is not configured.");
           }
 
           const {
             orgId,
-            pushNotifications: { vapidPublicKey } = {
+            pushNotifications: { vapidPublicKey, appId } = {
               vapidPublicKey: undefined,
+              appId: undefined,
             },
```

Comment:
You can just do `pushNotifications: { vapidPublicKey, appId } = {},`, but really you never use vapidPublicKey and appId except to pass along to makeSendPushSubscriptionRequest, so you don't need to spell out all the fields. You could just do `pushNotifications,`